src/main/sensors/battery.h

```diff
@@ -40,6 +40,9 @@ typedef struct batteryConfig_s {
     currentMeterSource_e currentMeterSource; // source of battery current meter used, either ADC, Virtual or ESC
     uint16_t batteryCapacity;               // mAh
 
+    // cells
+    uint8_t setBatteryCellCount;            // number of cells in battery, used for overwriting auto-detected cell count if someone has issues with it.
```

Comment:
Also there may be better names: `forceBatteryCellCount`, `overrideBatteryCellCount`, etc ... `set` is a bit ambiguous